dcos/marathon.py

```diff
@@ -22,10 +22,9 @@ def create_client(config=None):
         config = util.get_config()
 
     marathon_url = _get_marathon_url(config)
-    timeout = config.get('core.timeout', http.DEFAULT_TIMEOUT)
 
     logger.info('Creating marathon client with: %r', marathon_url)
-    return Client(marathon_url, timeout)
+    return Client(marathon_url)
```

Comment:
I commented on this in the already-merged #283 PR, but I don't understand why we complicated the Marathon interface with this.  We're only ever using the default (core.timeout) value, so there's no need to make it configurable at the Marathon level.